metrics_utility/automation_controller_billing/report/base.py

```diff
@@ -17,6 +17,8 @@ class Base:
     RED_COLOR_HEX = 'FF0000'
     LIGHT_BLUE_COLOR_HEX = 'd4eaf3'
     GREEN_COLOR_HEX = '92d050'
+    YELLOW_WARNING_COLOR_HEX = 'FFCC17'
```

Comment:
I probably did this, but here is a suggestion.
```suggestion
    YELLOW_COLOR_HEX = 'FFCC17'
```
or
```suggestion
WARNING_COLOR_HEX = 'FFCC17'
```

In either case I think we should be consistent in the naming since I expect we will use these colors more as we grow.  Additionally, I think the colors should match the colors and use cases that [Patternfly](https://www.patternfly.org/design-foundations/colors) uses